frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/block_action_option_button.dart

```diff
@@ -30,6 +30,10 @@ class BlockOptionButton extends StatelessWidget {
         return ColorOptionAction(
           editorState: editorState,
         );
+      } else if (e == OptionAction.align) {
+        return AlignOptionAction(
+          editorState: editorState,
+        );
```

Comment:
I'd consider refactoring this to a `switch` statement since each `if` branch returns a value. It'll be better since the analyzer will complain if there are missing case implementations.

Reply:
nice.